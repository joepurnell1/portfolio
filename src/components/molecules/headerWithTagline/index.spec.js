import React from 'react';
import renderer from 'react-test-renderer';
import HeaderWithTagline from './index';

describe('Header with Tagline component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <HeaderWithTagline
        headerText="Shakes me, makes me lighter"
        taglineText="Fearless on my breath"
        colour="#7A2D15"
      />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
