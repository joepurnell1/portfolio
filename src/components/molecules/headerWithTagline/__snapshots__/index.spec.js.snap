// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header with Tagline component should render default as expected 1`] = `
<div>
  <h1
    className="sc-bdVaJa dXLWSe"
  >
    Shakes me, makes me lighter
  </h1>
  <h4
    className="sc-bwzfXH bJBZAi"
  >
    Fearless on my breath
  </h4>
</div>
`;
